### PR TITLE
feat: [CDS-34104]: CA certificate field added for service account option in K8s connector

### PR DESCRIPTION
**Pull request by bhavya-sinha**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by chetanvekariya-harness — DISMISSED** [review]

**Comment by bhavya-sinha** [comment]
retrigger jest

**Comment by bhavya-sinha** [comment]
retrigger jest

**Review by avinashmadhwani02 — DISMISSED** [review]

File: src/modules/35-connectors/components/CreateConnector/K8sConnector/StepAuth/Stepk8ClusterDetails.tsx [hunk 1]
```diff
@@ -132,8 +132,9 @@ const RenderK8AuthForm: React.FC<FormikProps<KubeFormInterface> & { isEditMode:
       )
     case AuthTypes.SERVICE_ACCOUNT:
       return (
-        <Container width={'42%'}>
+        <Container className={css.formFieldWidth}>
           <SecretInput name={'serviceAccountToken'} label={getString('connectors.k8.serviceAccountToken')} />
+          <SecretInput name={'clientKeyCACertificate'} label={getString('connectors.k8.clientKeyCACertificate')} />
```
Comment (ishantmehtaharness) [inline src/modules/35-connectors/components/CreateConnector/K8sConnector/StepAuth/Stepk8ClusterDetails.tsx:137]: Do we want to add any validation for this field?

Comment (bhavya-sinha) [reply src/modules/35-connectors/components/CreateConnector/K8sConnector/StepAuth/Stepk8ClusterDetails.tsx:137]: No this is an optional field

**Review by YounixIO — APPROVED** [review]

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
trigger cypress

**Pull request merged**